internal/engine/filter/convert.go

```diff
@@ -228,13 +229,18 @@ func joinQueries(f1, f2 string) string {
 
 func joinStrings(escape bool, a ...string) string {
 	if escape {
-		return joinEscaped(a...)
+		b, _ := joinEscaped(a...)
```

Comment:
Here you are ignoring the error. You need to pass it up explicitly to the caller. The signature needs to be like this:

```
func joinStrings(escape bool, a ...string) (string, error) {
```

So that you can return the error to the caller.